geomagio/JSON/JSONWriter.py

```diff
@@ -0,0 +1,212 @@
+        dict['metadata']['intermagnet']['imo'] = OrderedDict()
+        dict['metadata']['intermagnet']['imo']['iaga_code'] = stats.station
+        if 'station_name' in stats:
+            dict['metadata']['intermagnet']['imo']['name'] = stats.station_name
+        coords = [None] * 3
+        if 'geodetic_longitude' in stats:
+            coords[0] = str(stats.geodetic_longitude)
+        if 'geodetic_latitude' in stats:
+            coords[1] = str(stats.geodetic_latitude)
+        if 'elevation' in stats:
+            coords[2] = str(stats.elevation)
+        dict['metadata']['intermagnet']['imo']['coordinates'] = coords
+        dict['metadata']['intermagnet']['reported_orientation'] = ''.join(channels)
+        if 'sensor_orientation' in stats:
+            dict['metadata']['intermagnet']['sensor_orientation'] = stats.sensor_orientation
+        if 'data_type' in stats:
+            dict['metadata']['intermagnet']['data_type'] = stats.data_type
+        if 'sampling_rate' in stats:
+            if stats.sampling_rate == 1. / 60.:
+                rate = 60
+            elif stats.sampling_rate == 1. / 3600.:
+                rate = 3600
+            elif stats.sampling_rate == 1. / 86400.:
+                rate = 86400
+            else:
+                rate = 1
+            dict['metadata']['intermagnet']['sampling_period'] = str(rate)
+        if 'sensor_sampling_rate' in stats:
+            dict['metadata']['intermagnet']['digital_sampling_rate'] = str(1 / stats.sensor_sampling_rate)
+        dict['metadata']['status'] = 200
+        dict['metadata']['generated'] = datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
+        self.dictionary = dict
+
+    def _format_times(self, timeseries, channels):
+        """format times for json file and update dictionary
+
+        Parameters
+        ----------
+        timeseries: obspy.core.stream
+            timeseries object with data to be written
+        channels: array_like
+            channels to be reported.
+        """
+        times = []
+        traces = [timeseries.select(channel=c)[0] for c in channels]
+        starttime = float(traces[0].stats.starttime)
+        delta = traces[0].stats.delta
+        for i in range(len(traces[0].data)):
+            times.append(self._format_time_string(
+                datetime.utcfromtimestamp(starttime + i * delta)))
+        self.dictionary['times'] = times
+
+    def _format_time_string(self, time):
+        """format one time.
+
+        Parameters
+        ----------
+        time : datetime
+            timestamp for values
+
+        Returns
+        -------
+        unicode
+            formatted time.
+        """
+        tt = time.timetuple()
+        return '{0.tm_year:0>4d}-{0.tm_mon:0>2d}-{0.tm_mday:0>2d}T' \
+                '{0.tm_hour:0>2d}:{0.tm_min:0>2d}:{0.tm_sec:0>2d}.{1:0>3d}Z'.format(
+                tt, int(time.microsecond / 1000))
+
+    def _format_data(self, timeseries, channels, stats):
```

Comment:
Update this method to return a `values` list, which can be stored in the overall response dictionary.